src/views/About/index.jsx

```diff
@@ -5,14 +5,15 @@ import contributors from './contributors.json';
 import origins from './origins.json';
 import versionUI from './version.json';
 import styles from './About.module.css';
+import styles2 from 'app.module.css';
 
 import { Typography, Grid, Button, Card, CardHeader, CardContent } from '@mui/material';
 
 const About = () => {
   const [versionAPI, setVersion] = useState(null);
 
   useEffect(() => {
-    versionService.getVersion().then(setVersion);
+    //versionService.getVersion().then(setVersion);
```

Comment:
If this isn't needed, please delete the whole hook.

Reply:
I was originally getting errors with this line after a pull, I just forgot to add it back.